### PR TITLE
Resolves CoarchyP-100078: Statements improvement

### PR DESCRIPTION
**Pull request by marwand**
Changes:

- Added Filter by process story/actor in activities dialog
- Added unconnect all activities button in activities dialog, 
- Show message & open activities dialog when trying to delete statement with connected activities
- Fix statments report link

### CODE REVIEW

File: screen/coarchy/cointernal/ValueStatements.xml [hunk 1]
```diff
@@ -323,7 +375,39 @@ along with this software (see the LICENSE.md file). If not, see
...
+                                            <field name="valueStatementId"><default-field><hidden /></default-field></field>
+                                            <field name="_openDialog" from="'ConnectedActivitiesDialog_'+valueStatementEntry_index"><default-field><display /></default-field></field>    
+                                            <field name="processStoryIds" from="processStoryIds"><default-field title="Filter By Process Story">
+                                                <drop-down allow-empty="true" allow-multiple="true">
+                                                    <list-options list="allProcessStoryActivityList" key="${processStoryId}" text="${name}" />                               
+                                                </drop-down>
+                                            </default-field></field>
+                                            <field name="actorName" from="actorName"><default-field title="Filter By Actor">
+                                                <text-line/>
```
Comment (acetousk) [inline screen/coarchy/cointernal/ValueStatements.xml]: This should be a drop-down of actors using dynamic-options to filter by process story selected like https://github.com/coarchy/coarchy/blob/894f9ba7770a4d2989ce92744a2284ba8e52808c/screen/coarchy/cointernal/Process/EditProcessStory.xml#L230. If a process story isn't selected, then it should allow searching all of them.

**Review by acetousk — CHANGES_REQUESTED** [review]
Overall this looks good thanks!

**Comment by acetousk** [comment]
In the ValueStatement screen's activities dialog, change connect to Add and disconnect to Remove. This will make it less cumbersome

**Comment by marwand** [comment]
> In the ValueStatement screen's activities dialog, change connect to Add and disconnect to Remove. This will make it less cumbersome

Do you also want to change the form list header title "Connected Activities" to "Added Activities"?
Otherwise, I believe your comments have been resolved, let me know!

**Pull request merged**